### PR TITLE
DROOLS-5665: DMN editor not saving changes in properties panel in Business Central

### PR DESCRIPTION
**Pull request by danielzhe**
**JIRA**: [DROOLS-5665](https://issues.redhat.com/browse/DROOLS-5665)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/definition/model/InformationItem.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public String getStunnerCategory() {
     public InformationItem copy() {
         final InformationItem clonedInformationItem = new InformationItem();
         clonedInformationItem.description = Optional.ofNullable(description).map(Description::copy).orElse(null);
-        clonedInformationItem.name = Optional.ofNullable(name).map(Name::copy).orElse(null);
+      //  clonedInformationItem.nameHolder = Optional.ofNullable(name).map(Name::copy).orElse(null);
```
Comment (jstastny-cz) [inline kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/definition/model/InformationItem.java:106]: remove commented code.

**Review by jstastny-cz — COMMENTED** [review]
jenkins do fdb

**Comment by jstastny-cz** [comment]
jenkins do fdb

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/definition/model/InformationItem.java (hunk 1 above)
Comment (danielzhe) [reply kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/definition/model/InformationItem.java]: Thank you! Change applied!

**Review by karreiro — APPROVED** [review]
Thank you, @danielzhe :-)

**Comment by karreiro** [comment]
Jenkins do fdb.

**Comment by karreiro** [comment]
@jstastny-cz Could you please take a look on this one? :-)

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by karreiro** [comment]
Jenkins do fdb.

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by karreiro** [comment]
Jenkins do fdb.

**Comment by danielzhe** [comment]
Jenkins do fdb

**Comment by karreiro** [comment]
@danielzhe I'm wondering if we maybe need to rebase this PR to get the fdb green. Wdyt? :-)

**Review by jstastny-cz — APPROVED** [review]
LGTM

**Pull request merged**